astropy/tests/plugins/repeat.py

```diff
@@ -1,11 +1,10 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """
-These plugins modify the behavior of py.test and are meant to be imported
-into conftest.py in the root directory.
+This plugin provides a command-line option controlling how many times tests
+should be repeated.
 """
 
 
-
 def pytest_addoption(parser):
 
     parser.addoption('--repeat', action='store',
```

Comment:
I think we could actually remove this plugin in favor of just having people use [pytest-repeat](https://pypi.python.org/pypi/pytest-repeat) if they want this functionality. I doubt many people know about the ``--repeat`` option so I think it's fine to remove without deprecation.

Reply:
Okay so should I remove it in this PR or do it separately after this is merged? Also if you or someone else can create `astropy/pytest-repeat` I can take care of setting it up and putting out an alpha release.

Reply:
@drdavella - just to be clear, pytest-repeat already exists! (see link in my previous comment). So I think you can just remove the code here and not worry about it.